bazarr/get_subtitle.py

```diff
@@ -822,6 +831,28 @@ def refine_from_db(path, video):
     return video
 
 
+def refine_from_mediainfo(path, video):
+    exe = get_binary('mediainfo')
+    if not exe:
+        logging.warn('BAZARR MediaInfo library not found!')
+        return
+
+    media_info = MediaInfo.parse(path, library_file=exe);
+
+    video_track = next((t for t in media_info.tracks if t.track_type == 'Video'), None)
+    if not video_track:
+        logging.warn('BAZARR MediaInfo was unable to find video tracks in the file!')
+        return
+
+    logging.debug('MediaInfo found: %s', video_track.to_data())
+
+    if not video.fps:
```

Comment:
Why not do this before analyzing the file to get the FPS? If the video object already include the FPS you don't use the result of MediaInfo analysing...

Reply:
Fair point, I'll move the check to the top.